README.md

```diff
@@ -17,13 +17,17 @@ git clone git@github.com:vuejs/docs.git
 2. Install dependencies
 
 ```bash
-yarn # or npm install
```

Comment:
I don't think you have to change this

Reply:
Just thought of making it clear
but it's okay